cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -56,6 +57,7 @@ func WriteKubeletDynamicEnvFile(nodeRegOpts *kubeadmapi.NodeRegistrationOptions,
 	flagOpts := kubeletFlagsOpts{
 		nodeRegOpts:              nodeRegOpts,
 		featureGates:             featureGates,
+		pauseImage:               pauseImage,
```

Comment:
i guess we can keep featureGates and pauseImage as part of kubeletFlagsOpts, but obtain them from a ClusterConfiguration argument.